3rd-party/ska/flat_hash_map.hpp

```diff
@@ -356,16 +356,7 @@ class sherwood_v3_table : private EntryAlloc, private Hasher, private Equal
         : EntryAlloc(alloc), Hasher(other), Equal(other), _max_load_factor(other._max_load_factor)
     {
         rehash_for_other_container(other);
-        try
-        {
-            insert(other.begin(), other.end());
-        }
-        catch(...)
-        {
-            clear();
-            deallocate_data(entries, num_slots_minus_one, max_lookups);
-            throw;
-        }
+        insert(other.begin(), other.end());
```

Comment:
Is it no longer possible for insert to throw?

Reply:
It could throw if the allocator throws, but at that point we'd be in pretty bad shape anyway, and very difficult to recover from that state. Better to die, and let our supervisor restart us.